package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "knockout-place-viewer",
```

Comment:
this one isn't in the webpack alias config. it's being required as `knockout-place-viewer`.